File: Beecrowd/2783 Figurinhas da Copa/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
    int n, c, m;

    cin >> n >> c >> m;

    int fc[c], fb[m];
    int cont = c;

    for (int i = 0; i < c; i++) {
        cin >> fc[i];
    }

    for (int i = 0; i < m; i++) {
        cin >> fb[i]; 
    }

    for (int i = 0; i < c; i++) {
        for (int j = 0; j < m; j++) {
            if (fb[j] == fc[i]) {
                cont--;
                break;
            }
        }
    }

    cout << cont << endl;

    return 0;
}

File: Beecrowd/2783 Figurinhas da Copa/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int n, c, m;

        n = sc.nextInt();
        c = sc.nextInt();
        m = sc.nextInt();

        int[] fc = new int[c];
        int[] fb = new int[m];
        int cont = c;

        for (int i = 0; i < c; i++) {
            fc[i] = sc.nextInt();
        }
    
        for (int i = 0; i < m; i++) {
            fb[i] = sc.nextInt();
        }
    
        for (int i = 0; i < c; i++) {
            for (int j = 0; j < m; j++) {
                if (fb[j] == fc[i]) {
                    cont--;
                    break;
                }
            }
        }
    
        System.out.println(cont);

        sc.close();
    }

}